### ISSUE TITLE
Useless without allowing for variables in command invocations

### ISSUE THREAD

**Issue by jcromartie** [sunng87/shake#1]
For example, you cannot do this:

```
(let [x "/usr/share"]
  (ls x))
```

It is really difficult to selectively interpret certain symbols this way.

**Comment by sunng87** [comment]
I'm looking for a way to inject clojure variable into the macro. Maybe it will look like (ls $x), any ideas?

**Comment by jcromartie** [comment]
$x would work, if you can do it, but I've not found a way to access the
value of a local binding inside a macro.

The basic approach would rely on the implicit `&env` variable available to
macros, but like I said, I don't know how to get at the actual value.

**Comment by sunng87** [comment]
Fixed and released in 0.2.0

Now you can use local bindings or vars in shake macros:

``` clojure
(let [x "/home/sunng87"]
  (ls -l $x))
```
